src/pykeen/pipeline/api.py

```diff
@@ -1011,6 +1011,7 @@ def pipeline(  # noqa: C901
     if dataset is not None:
         _result_tracker.log_params(
             dict(
+                # FIXME
```

Comment:
remove this fixme

Reply:
[ee22681](https://github.com/pykeen/pykeen/pull/569/commits/ee22681939cc83e7853c7644e171b9ac16aac6b3)